news/_posts/core-weekly/2019-04-19-coreweekly-week-15-2019.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 14 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-04-08 14:30:00
```

Comment:
We're the 19th of April are we not 🤔?

Reply:
header is updated, I was editing it while you wrote this comment.